cc/src/main/scala/me/sgrouples/rogue/BsonFormats.scala

```diff
@@ -177,6 +177,17 @@ trait BaseBsonFormats {
     override def defaultValue: Currency = Currency.getInstance("USD")
   }
 
+  implicit object LocaleBsonFormat extends BasicBsonFormat[Locale] {
+    override def read(b: BsonValue): Locale = {
+      //taken from lift's LocaleField as the simpliest solution, albeit not optimal.
+      //commons-lang LocaleUtils were not able to parse all Locale.getAvailableLocales either.
+      val value = b.asString().getValue
+      Locale.getAvailableLocales.filter(_.toString == value).headOption.getOrElse(defaultValue)
+    }
+    override def write(l: Locale): BsonValue = new BsonString(l.toString)
+    override def defaultValue: Locale = Locale.ENGLISH
```

Comment:
I am not sure about it: 
`  val defaultLocale: Locale = new Locale("en")` https://github.com/sgrouples/sgrouples/blob/master/utils/src/main/scala/me/sgrouples/util/Translation.scala#L26